aws-s3/latest/config/direct-access.md

```diff
@@ -0,0 +1,137 @@
+
+## REST API configuration
+
+The REST API configuration only affects the REST layer in Content Services.
+
+| Property | Description |
+| -------- | ----------- |
+| restApi.directAccessUrl.enabled | Enables/disables DAU requests via the REST API (for example `false`). |
+| restApi.directAccessUrl.defaultExpiryTimeInSec | Sets the expiry time in seconds for all the DAUs requested via a REST call (for example `30`). DAU REST API calls cannot request an explicit expiry time - unlike the service layer calls).<br><br>Its value cannot exceed the system-wide maximum expiry time configuration (`system.directAccessUrl.maxExpiryTimeInSec`) - it can only be equal to or lower (otherwise the REST API DAUs are disabled).<br><br>If it's not set, the default system-wide property is used (`system.directAccessUrl.defaultExpiryTimeInSec`). |
+
+## Storage connector content store
+
+In the example of the S3 Connector, each content store (i.e. "final" content store, one that provides actual storage, as opposed to a caching content store), should have dedicated configuration options:
+
+| Property | Description |
+| -------- | ----------- |
+| connector.s3.directAccessUrl.enabled | Controls whether DAUs are enabled on this specific content store (for example `false`). |
+| connector.s3.directAccessUrl.defaultExpiryTimeInSec | Sets the expiry time in seconds for the DAU in this store, by overriding the global configuration (for example `30`). <br><br>If this value exceeds the content store limit (described below) or the global limit it should fallback to the global configuration. Its value cannot exceed the system-wide maximum expiry time configuration (`system.directAccessUrl.maxExpiryTimeInSec`) - it can only be equal or lower (otherwise DAUs for the specific content store will be disabled). <br><br>If it's not set, the default system-wide setting is used (`system.directAccessUrl.defaultExpiryTimeInSec`). |
+| connector.s3.directAccessUrl.maxExpiryTimeInSec=300 | The maximum expiry time interval that can be requested by clients - content-store specific setting. <br><br>Its value cannot exceed the system-wide configuration (`system.directAccessUrl.maxExpiryTimeInSec`) - it can only be equal or lower (otherwise DAUs for the specific content store will be disabled). <br><br>If it's not set, the default system-wide setting is used (`system.directAccessUrl.maxExpiryTimeInSec`). |
```

Comment:
300 secs (5 mins) sounds too long - shouldn't this be 30 seconds ? See also below.